docs/docs/self-hosting/configuration/application.md

```
@@ -0,0 +1,194 @@
+---
+sidebar_position: 2
+---
+
+# Application
+
+Core application settings including database, timezone, proxy configuration, and logging.
+
+## Application Settings
+
+| Variable | Description | Default |
+|----------|-------------|---------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)